store/redis.go

```diff
@@ -1,22 +1,25 @@
 package store
 
 import (
+	"context"
 	"fmt"
 	"time"
 
-	redis "github.com/go-redis/redis/v7"
+	redis "github.com/go-redis/redis/v8"
 )
 
+var ctx = context.Background()
```

Comment:
I think context should be given from the cache entry point to the stores behind, not initialized like this.

Do you have some time to do it? In case you haven't time, we can merge it like it is and I will rework the interfaces to allow passing a context from the beginning later.